ports/esp32/modnetwork.c

```diff
@@ -34,6 +35,8 @@
 #include <stdint.h>
 #include <string.h>
 
+#include "network_lan.h"
```

Comment:
Please use modnetwork.h for a common header file.  Since the header code is not expected to excessive in size it's better to put all the data for WLAN/LAN/etc in modnetwork.h.

Reply:
Also, please move this include to after the core py/ includes.

Reply:
done